screens/org-skillimpact-screen/OrgSkillImpactScreen.js

```diff
@@ -25,23 +27,65 @@ class OrgSkillImpactScreen extends Component {
       user: {},
       scrollY: 0
     };
+    this.profileId =
+      this.props.navigation.getParam('selectedProfile') ||
+      this.props.currentUserProfile.id;
+    this.skills =  this.props.navigation.getParam('selectedProfile') ||
+      this.props.currentUserProfile.skills;
   }
 
-  componentDidMount() {
+
+  componentDidMount = () => {
     //TODO when the screen is loaded
-  }
+    this.initProfileData();
+    this._sub = this.props.navigation.addListener(
```

Comment:
**question**: Do we need to release this listener on unmount? If not, let's just get rid of the variable.